application/helpers/update/updates/Update_493.php

```diff
@@ -9,6 +9,7 @@ class Update_493 extends DatabaseUpdateBase
      */
     public function up()
     {
+        $this->db->createCommand()->addColumn('{{surveys_languagesettings}}', 'surveyls_alias', 'string(100)');
```

Comment:
We can not create a 494 for this update ?

I'm really for the minimum update between each number :)